content/applications/finance/fiscal_localizations/india.rst

```diff
@@ -348,6 +351,33 @@ fill out the :guilabel:`Cancel reason` and :guilabel:`Cancel remarks` fields. Th
      the government portal. You can click :guilabel:`Process Now` if you want to process the invoice
      immediately.
 
+.. _india/gstin_status:
+
+Indian Check GSTIN Status
+=========================
+
+The :guilabel:`Indian - Check GST Number Status` module allows you to verify the status of a GSTIN
+(Goods and Services Tax Identification Number) directly from Odoo.
+
+There are two ways to verify the status of a GST Number in Odoo:
+
+#. To verify the status of a partner's GST Number, go to :menuselection:`Accounting/Invoicing --> Customers/Vendors`
+   and select a partner. In the :guilabel:`GSTIN` field, click on the :guilabel:`Check GSTIN Status` button.
+
+   .. image:: india/gstin-status-partner.png
+      :alt: Check GSTIN status of a partner
+
+#. To verify the status of a GST Number entered in an invoice/bill, go to :menuselection:`Accounting/Invoicing --> Invoices/Vendor Bills`
+   and create a new invoice/bill and post the invoice/bill. In the :guilabel:`GST Status` field, click on the :guilabel:`GSTIN Status Refresh` button.
+
+   .. image:: india/gstin-status-invoice.png
+      :alt: Check GSTIN status of an invoice
```

Comment:
Do we necessarily need to *create* a new invoice/bill? Can't we check the status of existing invoices/bills? (sorry if this is a dumb question, and I can't test it at the moment)

```suggestion
To verify the status of a GST number entered on an invoice/bill, access the invoice/bill and click the :icon:`fa-refresh` (:guilabel:`refresh`) button next to the :guilabel:`GST Status` field.
```

Reply:
Not necessarily of a `new` invoice/bill. But the user can check the status until the full payment is paid (of the posted moves).